docs/MODULES.md

```diff
@@ -257,6 +257,8 @@ This module is used for decoding a QR in image (if present).
 
 ## dither-module
 
+[wikipedia-link] : "wikipedia-link":"https://en.wikipedia.org/wiki/Dither"
```

Comment:
Maybe instead of saying wikipedia-link, maybe we can say "more info" or "more info on [Wikipedia](https://link)" ?

Reply:
Also I think adding the link after the description would look better.

Also the way to add link is as follows

```
[Link-Text](https://link)
```
Will be changed to [Link-Text](https://link)

E.g.
```
[Google](https://google.com)
```
Will be changed to [Google](https://google.com)

Reply:
Thanks!! 
Done!

Reply:
I think the first one looks better? What do you say?

Reply:
The first one looks weird. The text is not a link at all, to add a proper link, only the link text should be visible in blue and it should link to the proper site so that the link text describes what website it leads to. I
n github md you can add links by using the syntax I commented above.

Reply:
Is it done? Or should I make should I make any more changes?